app/src/main/java/com/google/droidjump/GameConstants.java

```diff
@@ -25,8 +25,13 @@
     int DROID_FIRST_STEP_INDEX = 5; // Index of droid at res/mipmap/droid.png image.
     int DROID_SECOND_STEP_INDEX = 6; // Index of droid at res/mipmap/droid.png image.
     int SLEEP_TIME = 15; // Sleep time in gameView (ms).
-    int GAME_LEVELS_COUNT = 100; // Count of game levels.
-
-    // String for identifying current level from SharedPreferences.
-    String GAME_VIEW_LEVEL_STRING = "java.com.google.droidjump.GameView.Level";
+    int GAME_LEVELS_COUNT = 4; // Count of game levels.
+    int GAME_DEFAULT_VALUE = 1; // Default value.
+    String GAME_LEVEL_HEADER = "Lvl."; // Used for displaying the current level in GameView.
```

Comment:
nit: separate string variables from the int variables with a new line

Reply:
Fixed.